.github/workflows/playwright.yml

```diff
@@ -41,32 +41,17 @@ jobs:
     name: Build plugin
     uses: ./.github/workflows/build.yml
   Playwright:
-    name: Playwright test - ${{ matrix.testSuite }} on PHP 7.4
+    name: Playwright test - ${{ matrix.shardIndex }} on PHP 8.0
     runs-on: ubuntu-latest
     needs: [build-plugin]
     if: ${{ github.event.pull_request.title == null || needs.build-plugin.outputs.changelog_diff }}
     strategy:
-      fail-fast: ${{inputs.fail_fast || true}}
+      fail-fast: ${{ inputs.fail_fast || true }}
       matrix:
-        testSuite: [
-          'ai',
-          'onBoarding',
-          'video',
-          'elements-regression',
-          'default',
-          'nested-tabs',
-          'reverse-columns',
-          'container',
-          'nested-accordion',
-          'styleguide_image_link',
-          'rating',
-          'pluginTester1_containers',
-          'pluginTester2_containers',
-          'pluginTester1_sections',
-          'pluginTester2_sections',
-          'promotions',
-          'nested-atomic-repeaters',
-        ]
+        shardIndex: [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ]
+        shardTotal: [ 10 ]
+        include:
+        - shardIndex: "elements-regression"
```

Comment:
Are these lines needed? 
Aren't  `elements-regression` tests added by default like all the other tests?

Reply:
No - this means that on top of the shards, we have another scenario with this value